apis/src/lag_tracking/lag_tracker.rs

```diff
@@ -0,0 +1,103 @@
+use crate::lag_tracking::{decaying_stats::DecayingStats, stats::Stats};
+
+#[derive(Clone, Debug)]
+pub struct LagTracker {
+    pub quota_gain: f64,
+    pub quota: f64,
+    pub quota_max: f64,
+    pub lag_estimator: DecayingStats,
+    pub uncomp_stats: Stats,
+    pub lag_stats: Stats,
+    pub comp_est_sq_err: f64,
+    pub comp_est_overs: f64,
```

Comment:
I think that those two values are not really used to calculate the compensation in the end